GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -12,7 +12,7 @@ namespace GVFS.Platform.Mac
     public partial class MacPlatform : GVFSPlatform
     {
         public MacPlatform()
-            : base(executableExtension: string.Empty)
+            : base(executableExtension: string.Empty, installerExtension: ".dmg")

```

Comment:
Shouldn't this be using the `InstallerExtension` const?

Reply:
Added to the cleanup tasks.